iroh/src/endpoint.rs

```diff
@@ -417,6 +423,14 @@ impl Builder {
         self.insecure_skip_relay_cert_verify = skip_verify;
         self
     }
+
+    /// This implies we only use the relay to communicate
+    /// and do not attempt to do any hole punching.
+    #[cfg(any(test, feature = "test-utils"))]
+    pub fn relay_only(mut self, relay_only: bool) -> Self {
```

Comment:
It is not the end of the world since this is a debug feature, but I would probably favour an enum to configure which path to use instead of a bool. E.g. we might in the future want to have more complex logic to select which path to take (IPV4/IPV6/Relay)

Reply:
I guess that will make more sense to implement once we start dealing with multipath as we currently don't really have an idea of everything we will need to pass in.

Reply:
Make it an enum with 2 cases, then it is still easier to extend than a bool